tests/algorithms/test_preference_comparisons.py

```diff
@@ -245,3 +245,21 @@ def test_store_and_load_preference_dataset(agent_trainer, fragmenter, tmp_path):
 
         assert preference == loaded_preference
         _check_trajs_equal(fragments, loaded_fragments)
+
```

Comment:
Weak preference but I'd consider adding one or two more tests of `ExplorationWrapper`, mainly because it actually seems amenable to unit testing. Ideas:
  1. If `random_prob == 0`, is `self.current_policy` always `wrapped_policy`, and if `== 1`, always `random_policy`? If `0.5`, is it (roughly) equal after ~100 calls to switch? Can make this deterministic by setting seed.
  2. Check `__call__` always returns things inside `venv.action_space`

Reply:
Done